mod/message.php

```diff
@@ -148,7 +148,7 @@ function message_content(App $a)
 
 		$cmd = $a->argv[1];
 		if ($cmd === 'drop') {
-			if (dba::delete('mail', ['id' => $a->argv[2], 'uid' => local_user()])) {
+			if (dba::delete('mail', ['id' => $a->argv[2]])) {
```

Comment:
Don't worry, this has been brought back to live with the final commit of this PR.